src/scenes/forgotPassword/ForgotPassword.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import './forgotPassword.css';
+import { strings } from '../../utils';
+import { BasicInput, Button } from '../../components';
+import { forgotPassword } from '../../redux/auth/actions';
+import constants from '../../redux/auth/constants';
+import { regexes } from '../../constants';
+
+class ForgotPassword extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: { value: '', isValid: true, errorMessage: strings.LOGIN_EMAIL_REQUIRED },
```

Comment:
Is it needed to use the email object? Jasper told me it's best to keep the state as flat as possible.

Reply:
you mean: `this.state = {email: '', emailIsValid: true, errorMessage: ''}` ?